chia/cmds/cmds_util.py

```diff
@@ -103,13 +111,15 @@ async def get_any_service_client(
         # check if we can connect to node, and if we can then validate
         # fingerprint access, otherwise return fingerprint and shutdown client
         fingerprint = await validate_client_connection(
-            node_client, node_type, rpc_port, root_path, fingerprint, login_to_wallet
+            node_client, node_type, rpc_port, root_path, fingerprint, login_to_wallet, consume_errors
         )
         if node_client.session.closed:
             yield None, config, fingerprint
         else:
             yield node_client, config, fingerprint
     except Exception as e:  # this is only here to make the errors more user-friendly.
+        if not consume_errors:
+            raise
```

Comment:
Should this be after the exception?  I'm not entirely sure, but logging here does provide the extra context of the node type.

Reply:
nope, we skip this as its for the cli.